src/main/java/it/pagopa/pn/delivery/svc/search/NotificationRetrieverService.java

```diff
@@ -204,8 +204,8 @@ private void validateInput(InputSearchNotificationDto searchDto) {
 	 * @return Notification DTO
 	 *
 	 */
-	public InternalNotification getNotificationInformation(String iun, boolean withTimeline) {
-		log.debug( "Retrieve notification by iun={} withTimeline={} START", iun, withTimeline );
+	public InternalNotification getNotificationInformation(String iun, boolean withTimeline, boolean sender) {
```

Comment:
Utilizziamo un nome più esplicativo e metterei anche un javadoc per spiegare bene questo parametro (in più agli altri) perché non è immediato.
Potremmo chiamarlo senderSide, senderVersion, requestBySender ma sender solo non è chiaro. 
A me piace requestBySender